app/controllers/cash_transactions_controller.rb

```diff
@@ -0,0 +1,63 @@
+class CashTransactionsController < ApplicationController
+  resource_description do
+    short 'Add cash transactions to an assessment'
+    formats ['json'] # rubocop:disable Layout/SpaceBeforeBrackets
+    description <<-END_OF_TEXT
+    == Description
+      Adds cash income and outgoings to and assessment so that they can be included in the means test.
+
+    END_OF_TEXT
+  end
+  api :POST, 'assessments/:assessment_id/cash_transactions', 'Add cash income and outgoings'
+  formats ['json'] # rubocop:disable Layout/SpaceBeforeBrackets
+  param :assessment_id, :uuid, required: true
+  param :income, Array, of: Hash, desc: 'Collection of Income categories' do
+    param :category, CFEConstants::VALID_INCOME_CATEGORIES, required: true, desc: 'An identifying name for this income category'
+    param :payments, Array, of: Hash, desc: 'Total payments received for this category in each of the three months preceding the assessment date' do
+      param :date, Date, date_option: :today_or_older, required: true,
+                         desc: 'The date payment received (this must be the first day of one of the three months preceeding the assessment date'
+      param :amount, :currency, currency_option: :not_negative, required: true, desc: 'Amount of payment'
+      param :client_id, String, required: true, desc: 'Uniquely identifying string from client'
+    end
+  end
+  param :outgoings, Array, of: Hash, desc: 'Collection of Outgoing categories' do
+    param :category, CFEConstants::VALID_OUTGOING_CATEGORIES, required: true, desc: 'An identifying name for this outgoing category'
+    param :payments, Array, of: Hash, desc: 'Total payments made for this category in each of the three months preceding the assessment date' do
+      param :date, Date, date_option: :today_or_older, required: true,
+                         desc: 'The date payment made (this must be the first day of one of the three months preceeding the assessment date'
+      param :amount, :currency, currency_option: :not_negative, required: true, desc: 'Amount of payment'
```

Comment:
if it's an outgoing payment, shouldn't it allow negative payments? Instead it shouldn't be more than zero?
or at least not  have that currency_option validation

It depends on how we want the data if we want outgoings: { payment: -100.00 } or  outgoings: { payment: 100 }